### PR TITLE
Refactored third-party modules import and fixed few minor bugs

### PR DESCRIPTION
**Pull request by SuperPaintman**
Good afternoon!

I changed the build system a little:
  * Removed third-party modules' files and moved them to the Node modules (`vivus` and `dom-to-image`)

Also I fixed few bugs (mistakes):
  * `innerText` -> `textContent`
  * `classList` -> `className` - because you assigned string to the `classList`

### CODE REVIEW

**Comment by octref** [comment]
Still, if we can just have a few kbs of minified files why have dependencies pulling everything in?

Other than that this looks good. Thanks!

**Comment by SuperPaintman** [comment]
> Still, if we can just have a few kbs of minified files why have dependencies pulling everything in?

The first reason is that they look like we want to hide themfrom users, like a virus or other "sick" code.

The second reason it that VS Code pull the dependencies on its own (on install). And there is no reason to store them in this repo.

**Comment by sangeeth96** [comment]
I agree with @SuperPaintman here. I thought there was magic thing going on in `dom2image.js` and besides that, if it gets updated in the future with new features/changes, we'd just have to flip the versions in `package.json` and have it up & running.

File: src/webview/index.js [hunk 1]
```diff
@@ -89,7 +89,9 @@ function stripInitialIndent(html, indent) {
   const doc = new DOMParser().parseFromString(html, 'text/html')
   const initialSpans = doc.querySelectorAll('div > div span:first-child')
   for (let i = 0; i < initialSpans.length; i++) {
-    initialSpans[i].innerText = initialSpans[i].innerText.slice(indent)
+    const initialSpan = initialSpans[i]
```
Comment (octref) [inline src/webview/index.js]: just one line is fine. don't declare unnecessary variables.

**Comment by octref** [comment]
Well, I went for #40. This is my repo so I do what I like.

If you can limit this PR to just the `textContent`, `className` and opacity change I'll take it. Otherwise no.

**Comment by SuperPaintman** [comment]
> Well, I went for #40. This is my repo so I do what I like.

Sure, It's totally your choice. If you want to confuse users with minified files - you can do it.


> If you can limit this PR to just the textContent, className and opacity change I'll take it. Otherwise no.

Chould you do it on your own?

**Pull request closed without merging**